libunwind/src/UnwindRustSgx.h

```diff
@@ -28,22 +28,8 @@ typedef struct {
 } RWLock;
```

Comment:
`unsigned char opaque[128];` 
This size might change with new implementation, mostly to 16.

Reply:
Oh, yeah. It not only might change. It will change to 16. Or more exactly `size_of::<usize>() * 2`. But since SGX is 64 bit only that will be 16, indeed.

The comment for this struct also contains the old path, that won't exist any longer: `rwlock.rs`

Reply:
Maybe not that important, but the new type is called `RwLock` and not `RWLock` also. But as long as the memory layout is correct, the naming does not have to be the same on both sides.